src/com/teamstu/Main.java

```diff
@@ -4,5 +4,6 @@
 
     public static void main(String[] args) {
         System.out.println("Hello World!");
+        System.out.println("Hello you!");

```

Comment:
Hi Mark, this looks good but could you please make this "Hello Yourself", thanks.